### ISSUE TITLE
Small case left out by clippy::match_like_matches_macro

### ISSUE THREAD

**Issue by kamulos** [rust-lang/rust-clippy#6346]
Just a suggestion for a small improvement:

This code gets a suggestion from clippy::match_like_matches_macro
```
    match &args[1][..] {
        "--version" => true,
        _ => false,
    }
```

While this one does not, while it still benefits from being converted to the matches macro:

```
    match &args[1][..] {
        "-v" => true,
        "--version" => true,
        _ => false,
    }
```

**Comment by giraffate** [comment]
Could you provide an example on playground? I tried [this on playground](https://play.rust-lang.org/?version=nightly&mode=debug&edition=2018&gist=14985f229428fa6d80ed34459529db28) but the lint was emitted.

**Comment by giraffate** [comment]
https://github.com/rust-lang/rust-clippy/pull/6216 may have fixed this case. I think this would be released at next beta release(Rust 1.49).

**Comment by flip1995** [comment]
Yes, I remember putting this in the changelog for beta today. [This is linted in nightly (and soon on beta)](https://play.rust-lang.org/?version=nightly&mode=debug&edition=2018&gist=030cc927b09fa6739c7beb7cf5e0be30)
